kafka_utils/kafka_check/commands/command.py

```diff
@@ -80,6 +90,14 @@ def is_controller(zk, broker_id):
     return broker_id == zk.get_json('/controller').get('brokerid')
 
 
-def is_first_broker(zk, broker_id):
+def get_broker_ids(zk):
+    """Returns a list of integers for broker ids"""
+    return list(zk.get_brokers())
+
+
+def is_first_broker(broker_ids, broker_id):
     """Returns true if broker_id is the lowest broker id in the cluster, false otherwise."""
-    return broker_id == min(zk.get_brokers().keys())
+    if not broker_ids:
```

Comment:
This can probably just be `broker_ids and broker_id == min(broker_ids)` 🙂

Reply:
Python is a bit weird. I tried `[] and 4 == min([])` and got empty list `[]` rather than `False` Python's logical operators seem to do something inconsistent. We have to around it using `if` statement.

Reply:
Oh. yeah. Python.

`len(broker_ids) > 0 and broker_id == min(zk.get_brokers().keys())` maybe? 😄